docs/userguide/networking/default_network/container-communication.md

```diff
@@ -72,20 +72,9 @@ Whether two containers can communicate is governed, at the operating system leve
 
 - Do your `iptables` allow this particular connection? Docker will never make changes to your system `iptables` rules if you set `--iptables=false` when the daemon starts.  Otherwise the Docker server will add a default rule to the `FORWARD` chain with a blanket `ACCEPT` policy if you retain the default `--icc=true`, or else will set the policy to `DROP` if `--icc=false`.
 
-It is a strategic question whether to leave `--icc=true` or change it to
-`--icc=false` so that `iptables` will protect other containers -- and the main
-host -- from having arbitrary ports probed or accessed by a container that gets
-compromised.
-
-If you choose the most secure setting of `--icc=false`, then how can containers
-communicate in those cases where you _want_ them to provide each other services?
-The answer is the `--link=CONTAINER_NAME_or_ID:ALIAS` option, which was
-mentioned in the previous section because of its effect upon name services.  If
-the Docker daemon is running with both `--icc=false` and `--iptables=true`
-then, when it sees `docker run` invoked with the `--link=` option, the Docker
-server will insert a pair of `iptables` `ACCEPT` rules so that the new
-container can connect to the ports exposed by the other container -- the ports
-that it mentioned in the `EXPOSE` lines of its `Dockerfile`.  
+It is a strategic question whether to leave `--icc=true` or change it to `--icc=false` so that `iptables` will protect other containers -- and the main host -- from having arbitrary ports probed or accessed by a container that gets compromised.
```

Comment:
can you wrap these lines to 80-chars again? We try to keep the Markdown source wrapped to 80 characters